students/minchan/westagram/users/views.py

```diff
@@ -1,3 +1,47 @@
-from django.shortcuts import render
+import json
+import re
+from django.http.response   import JsonResponse
+from django.views           import View
+from users.models           import Users
+from django.http            import HttpResponse
 
 # Create your views here.
+class UserRegister(View) :
+    REGEX_EMAIL     = re.compile("[@][a-zA-Z]*[.]")
+    REGEX_PASSWORD  = re.compile("/^(?=.*[a-zA-Z])(?=.*\d)(?=.*[`~!@#$%^&*(),<.>/?]).{8,}")
+
+    def valid_email(self,email) :
+        return True if self.REGEX_EMAIL.match(email) else False
+
+    def is_email_duplicate(self,email) :
+        return Users.objects.filter(email=email).exists()
+    
+    def valid_pw(self,pw) :
+        return True if self.REGEX_PASSWORD.match(pw) else False
+
+    def post(self,request) :
+        data    = json.loads(request.body)
+        email   = data["email"]
+        pw      = data["password"]
+        #이메일, 패스워드의 유효성 및 중복에 대해 검사.        
+        if not (email or pw) :
+            return JsonResponse({"MESSAGE": "KEY_ERROR"}, status=401)
+
+        if not self.valid_email(email) :
+            return JsonResponse({"MESSAGE" : "이메일 양식이 틀렸습니다."}, status=401)
```

Comment:
함수명은 동사로 시작하도록 수정해주세요~
```suggestion
        if not self.validate_email(email) :
            return JsonResponse({"MESSAGE" : "이메일 양식이 틀렸습니다."}, status=401)
```

Reply:
변경 완료 했습니다
(is_email_valid)

Reply:
민찬님 file change에서 확인해보면 반영이 안되어 있네요!?
commit 내역 한번 확인해 주실 수 있나요?